toolkit.rb

```diff
@@ -11,7 +11,7 @@
 args = {}
 args_array.each do |arg|
   arg_name  = arg.split("=").first.to_sym
-  arg_value = arg.split("=").last
+  arg_value = arg.gsub(arg.split("=").first, "").delete_prefix("=")
```

Comment:
Thank you for finding and fixing this issue with multiple equal signs!

Unfortunately, this is a dicey one to fix, and I'm afraid that even the solution has its own unique problems.  For instance, if the unparsed `arg` in this case were the string `foo=foo`, then this resulting substitution would produce an empty string.

I think what we should ultimately do here is optionally support quoted-strings, so that argument values including quotes and spaces can be honored appropriately.

You've done a good job trying to improve our homemade command-line parsing code here (thank you!), but in order to do this parsing safely, I believe that switching to a supported library may be a good approach here.  For instance, if we move to the Ruby standard library [OptionParser](https://ruby-doc.org/stdlib-2.6.6/libdoc/optparse/rdoc/OptionParser.html), it handles a lot of the complexity for us here for free.

Reply:
No worries, let me know what you decide to go with and we'll fit in - thanks!

Reply:
I think quoting the inbound params works. That is what we have done with other tasks where the keys have = signs in the key.

Reply:
@davidaustinarcher can you back this out and retest with quotes around the keys at run? @davidaustinarcher